frontend/src/TooljetDatabase/Forms/TableSchema.jsx

```diff
@@ -359,36 +371,48 @@ function TableSchema({
                 show={columnDetails[index]?.data_type === 'serial'}
               >
                 <div className="m-0" data-cy="column-default-input-field">
-                  <input
-                    onChange={(e) => {
-                      e.persist();
-                      const prevColumns = { ...columnDetails };
-                      prevColumns[index].column_default = e.target.value;
-                      setColumns(prevColumns);
-                    }}
-                    value={
-                      columnDetails[index].data_type === 'serial'
-                        ? 'Auto-generated'
-                        : // : checkDefaultValue(columnDetails[index].column_default)
-                          // ? null
-                          columnDetails[index].column_default
-                    }
-                    type="text"
-                    className="form-control defaultValue"
-                    data-cy="default-input-field"
-                    placeholder={
-                      (columnDetails[index].data_type === 'serial' &&
-                        columnDetails[index]?.constraints_type?.is_primary_key === true) ||
-                      columnDetails[index].data_type === 'serial'
-                        ? 'Auto-generated'
-                        : 'Enter value'
-                    }
-                    disabled={
-                      (columnDetails[index].data_type === 'serial' &&
-                        columnDetails[index]?.constraints_type?.is_primary_key === true) ||
-                      columnDetails[index].data_type === 'serial'
-                    }
-                  />
+                  {columnDetails[index].data_type === 'timestamp with time zone' ? (
+                    <DateTimePicker
+                      timestamp={columnDetails[index].column_default}
+                      timezone={columnDetails[index]?.configurations?.timezone || getLocalTimeZone()}
```

Comment:
This will always be in the local timezone on table creation, but on edit, it will be in the configured timezone.
How would the user know what TZ is being picked from UI? 🤔 
![image](https://github.com/ToolJet/ToolJet/assets/11629675/80ceee93-6b93-487f-b305-27179e3b9017)

Reply:
This by default picks the local timezone but if you click on the triple dots you will be able to change the timezone and see your current timezone as well.